app/routes/book_routes.py

```diff
@@ -21,18 +21,23 @@ def create_book():
     }
     return make_response(response, 201)
 
-# @books_bp.get("")
-# def get_all_books():
-#     books_response = []
-#     for book in books:
-#         books_response.append(
-#             {
-#                 "id": book.id,
-#                 "title": book.title,
-#                 "description": book.description
-#             }
-#         )
-#     return books_response
+@books_bp.get("")
+def get_all_books():
+    query = db.select(Book).order_by(Book.id)
```

Comment:
👍 Thank you for including the order by right from the start.

Reply:
I like that you went with building the query on its own rather than cramming it into the execution method parameter directly. This feels a little less overwhelming.